### PR TITLE
Twin change events forwarding

### PR DESCRIPTION
**Pull request by marcschier**
* Hook up serializers and VariantValue replacement for JObject/dynamic to support "value serialization"
* Switch models to non-Newtonsoft DataContract / DataMember attributes
* 1 set of API models and 1 set of core models, 1 conversion library (ModelExtension.cs) instead of multiple copies in the tree.
* Single SignalR endpoint instead of multiple across all services + unit tests to make signalr service optional.
* Message pack serialization tests for all API models and endpoint formatters
* Remove non-essential services and consolidate all IoTHub "daemons" into a single one (per hub).
* Several fixes as per internal VSO backlog.

### CODE REVIEW

File: Industrial-IoT.sln [hunk 1]
```diff
@@ -258,7 +234,15 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Serial
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Serializers.MessagePack.Tests", "common\src\Microsoft.Azure.IIoT.Serializers.MessagePack\tests\Microsoft.Azure.IIoT.Serializers.MessagePack.Tests.csproj", "{01A2DD25-32E4-424C-82E8-9C7111C64AAE}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Test.Scenarios", "api\src\Microsoft.Azure.IIoT.Api\tests\Microsoft.Azure.IIoT.Test.Scenarios.csproj", "{5B81D952-2E41-4704-8D22-2D817D5F94E6}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.AspNetCore.SignalR", "common\src\Microsoft.Azure.IIoT.AspNetCore.SignalR\src\Microsoft.Azure.IIoT.AspNetCore.SignalR.csproj", "{CF79D44A-E79C-42D6-BDCA-1DDFD7108AB5}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Services.OpcUa.Events.Tests", "services\src\Microsoft.Azure.IIoT.Services.OpcUa.Events\tests\Microsoft.Azure.IIoT.Services.OpcUa.Events.Tests.csproj", "{6CFFAE7C-5943-4954-B8BF-9C455AF7F071}"
+EndProject
```
Comment (mregen) [inline Industrial-IoT.sln:240]: too many files for codeflow :-( I don't see the changed files in /services...

Comment (marcschier) [reply Industrial-IoT.sln:240]: Codeflow works just fine.

Comment (mregen) [reply Industrial-IoT.sln:240]: which version do you have? I see only files under api/src

---
In reply to: [404276365](https://github.com/Azure/Industrial-IoT/pull/375#discussion_r404276365) [](ancestors = 404276365)

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/DefaultSessionManager.cs [hunk 2]
```diff
@@ -266,11 +283,13 @@ public class DefaultSessionManager : ISessionManager {
                 IEnumerable<EndpointDescription> filteredEndpoints = endpoints.ToArray();
```
Comment (mregen) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/DefaultSessionManager.cs:284]: should filter here if endpoint security policy is in list of supportedpolicies (in Opc.Ua.SecurityPolicies) or unsupported policy can be seletced

Comment (marcschier) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/DefaultSessionManager.cs:284]: Don't you have a branch with that particular change?  Can we pull it from there?

Comment (marcschier) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/DefaultSessionManager.cs:284]: Put on backlog and will fix outside of this PR.

**Pull request merged**